dashboard/src/components/ErrorAlert/PermissionsErrorAlert.tsx

```diff
@@ -26,12 +27,20 @@ class PermissionsErrorPage extends React.Component<IPermissionsErrorPage> {
         showGenericMessage={false}
       >
         <div>
-          <p>Ask your administrator for the following RBAC roles:</p>
-          <ul className="error__permisions-list">
-            {roles.map((r, i) => (
-              <PermissionsListItem key={i} namespace={namespace} role={r} />
-            ))}
-          </ul>
+          <p>Ask your administrator for the following permissions:</p>
```

Comment:
I'd put this `<p>` inside the following condition, so that if roles are defined, we say the current msg, but otherwise, if displaying the raw content, we say something more appropriate, such as "The following error was returned by helm:"

Reply:
I've outlined this in https://github.com/kubeapps/kubeapps/pull/1453

Reply:
:+1: I have added `The following error was returned:` (without `helm`) because this component can be rendered with something else (for example, the service catalog).